web/tutorial-actions-executor/e2e-tests/generate-app.test.ts

```diff
@@ -0,0 +1,70 @@
+import path from "path";
+import { describe, expect, it } from "vitest";
+import { $, fs, glob } from "zx";
+
+const TEST_APP_NAME = "e2e-test-app";
+const TEST_OUTPUT_DIR = "./e2e-tests/.result";
+const TEST_TUTORIAL_DIR = "./e2e-tests/fixtures/tutorial";
+const WASP_CLI_COMMAND = process.env.WASP_CLI_COMMAND ?? "wasp-cli";
+
+describe("generate-app e2e", () => {
+  it(
+    "should generate app from tutorial actions and match snapshots",
+    async () => {
+      const generateAppOptions = [
+        "--app-name",
+        TEST_APP_NAME,
+        "--output-dir",
+        TEST_OUTPUT_DIR,
+        "--tutorial-dir",
+        TEST_TUTORIAL_DIR,
+        "--wasp-cli-command",
+        WASP_CLI_COMMAND,
+      ];
+      await $`npm run tacte -- generate-app ${generateAppOptions}`.verbose();
+
+      const projectDirPath = path.join(TEST_OUTPUT_DIR, TEST_APP_NAME);
+
+      function getProjectFileContent(filePath: string): string {
+        return fs.readFileSync(path.join(projectDirPath, filePath), "utf-8");
+      }
+
+      const fileList = await getProjectFileList(projectDirPath);
+      expect(fileList).toMatchSnapshot("file-structure");
+
+      const testUtilsContent = getProjectFileContent("src/testUtils.ts");
+      expect(testUtilsContent).toMatchSnapshot("test-utils");
+
+      const schemaContent = getProjectFileContent("schema.prisma");
+      expect(schemaContent).toMatchSnapshot("prisma-schema");
+
+      const gitLog = await getGitLog(projectDirPath);
+      expect(gitLog).toMatchSnapshot("git-commits");
```

Comment:
Small thing, but for readibility, I would reduce this to:
```suggestion
      expect(await getProjectFileList(projectDirPath)).toMatchSnapshot("file-structure");
      expect(getProjectFileContent("src/testUtils.ts")).toMatchSnapshot("test-utils");
      expect(getProjectFileContent("schema.prisma")).toMatchSnapshot("prisma-schema");
      expect(await getGitLog(projectDirPath)).toMatchSnapshot("git-commits");
```

Less repetition, less boilerplate for me to read, faster to scan.